models/actions/run_list.go

```diff
@@ -101,6 +106,18 @@ func (opts FindRunOptions) ToConds() builder.Cond {
 	if opts.CommitSHA != "" {
 		cond = cond.And(builder.Eq{"`action_run`.commit_sha": opts.CommitSHA})
 	}
+	if !opts.CreatedAfter.IsZero() {
+		cond = cond.And(builder.Gte{"`action_run`.created": opts.CreatedAfter})
+	}
+	if !opts.CreatedBefore.IsZero() {
+		cond = cond.And(builder.Lte{"`action_run`.created": opts.CreatedBefore})
+	}
+	if opts.ExcludePullRequests {
+		cond = cond.And(builder.Neq{"`action_run`.trigger_event": webhook_module.HookEventPullRequest})
+	}
+	if opts.CheckSuiteID > 0 {
+		cond = cond.And(builder.Eq{"`action_run`.check_suite_id": opts.CheckSuiteID})
```

Comment:
There is no such column at the moment.

Reply:
So, take that out, that small part of the GitHub API compatibility won't exist, but it's a pretty minor aspect, I think?

Reply:
It seem that this PR is written by AI, and the non-existing column is caused by AI hallucination?

We need to make sure every line you proposed to change should be fully understood and has a clear meaning, I think?

Reply:
Yeah, yeah - for sure. So, remove it and leave a comment somewhere ... noticeable that the API is compatible except for this field. I'll get to work on that.

Reply:
OK, I think I got a little screwy with the commits, but all the changes seem to be there again. I left the `check_suite_id` in the swagger with a description that it's not supported - does that seem appropriate?